anvil/src/shell/mod.rs

```diff
@@ -152,6 +152,22 @@ impl<BackendData: Backend> CompositorHandler for AnvilState<BackendData> {
             }
             if let Some(window) = self.window_for_surface(&root) {
                 window.0.on_commit();
+
+                if &root == surface {
```

Comment:
For subsurfaces it is supposed to do nothing?

Reply:
That's not so easy to answer, it might have been intended to also apply to subsurfaces. But as mutter and wlroots always ignored the offset for subsurface [weston has been changed](https://gitlab.freedesktop.org/wayland/wayland/-/issues/389) to also ignore it. So imo we should follow that and just accept this as the common behavior.